conductr_cli/sandbox_features.py

```diff
@@ -468,16 +548,23 @@ def collect_features(feature_args, no_default_features, image_version, offline_m
         features.append(feature)
 
     def add_default_features(features):
+        names = [feature.name for feature in features]
+
         def add_proxying(features):
-            features.insert(0, feature_lookup[ProxyingFeature.name]([], image_version, offline_mode))
+            if ProxyingFeature.name not in names:
+                features.insert(0, feature_lookup[ProxyingFeature.name]([], image_version, offline_mode))
 
         def add_logging_lite(features):
-            names = [feature.name for feature in features]
             if LoggingFeature.name not in names:
                 features.insert(0, feature_lookup[LiteLoggingFeature.name]([], image_version, offline_mode))
 
+        def add_oci_in_docker(features):
+            if OciInDockerFeature.name not in names and not host.is_linux():
+                features.insert(0, feature_lookup[OciInDockerFeature.name]([], image_version, offline_mode))
+
```

Comment:
I wonder if there is a use-case where we don't want to run OCI images... perhaps not as we want almost everything to run via OCI.